app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -513,10 +513,41 @@ bool OpenXRInputSource::GetHandTrackingInfo(const XrFrameState& frameState, XrSp
         if (mHasAimState)
             mHandAimPose = aimState.aimPose;
     } else {
-        auto aimJoint = jointLocations.jointLocations[HAND_JOINT_FOR_AIM];
-        mHasAimState = IsHandJointPositionValid(HAND_JOINT_FOR_AIM);
-        if (mHasAimState)
-            mHandAimPose = aimJoint.pose;
+        mHasAimState = IsHandJointPositionValid(XR_HAND_JOINT_WRIST_EXT);
+        if (mHasAimState) {
+            mHandAimPose = jointLocations.jointLocations[XR_HAND_JOINT_WRIST_EXT].pose;
```

Comment:
My proposal is to use `XR_HAND_JOINT_MIDDLE_PROXIMAL_EXT` instead (the knuckle of the middle finger) so we also take the general position of the hand into account.

Reply:
Looks like a great suggestion